pkg/store/consul.go

```diff
@@ -95,11 +95,13 @@ func (s *Consul) setEphemeralTTL(ttl time.Duration) error {
 	return nil
 }
 
-// CreateEphemeralSession creates the a global session
+// CreateEphemeralSession creates the global session
```

Comment:
@aluzzardi I agree for consistency, but this method is unexported. So golint won't complain